test/specs/member-e2e.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-undef */
+const SignUpMember = require("../../test/pageObjects/member/signUpPage.js");
+const Buttons = require ("../../test/pageObjects/sharedComponents/button.js");
+const ModalAlert = require ("../../test/pageObjects/sharedComponents/modalAlert.js");
+const ModalConfirm = require ("../../test/pageObjects/sharedComponents/modalConfirm.js");
+const LogIn = require("../../test/pageObjects/sharedComponents/login.js");
+const modalAlert = require("../../test/pageObjects/sharedComponents/modalAlert.js");
+
+describe('page of sign up', () => {
+  beforeAll('open browser', () => {
+    browser.url('https://juvi-megarocket-app.vercel.app/auth');
+  });
+
+  it ('complete form', async () =>{
+    await SignUpMember.SingUpNavbarClick();
+    await expect(browser).toHaveUrl('https://juvi-megarocket-app.vercel.app/auth/sign-up')
+    await SignUpMember.formSignUp("Victoria", "Ramirez", "vicrami25@gmail.com", "vicrami25@gmail.com", "31665222", "3415568223",
+     "Fisherton", "19-05-1995", "2000", "vicram2514");
+    await SignUpMember.optionSelectBlackClick();
+    await browser.pause(2000);
+    await Buttons.submitBtnClick();
+    await expect(ModalAlert.modalAlertText).toBeDisplayed();
+    await ModalAlert.modalAlertButtonClick();
+    await expect(browser).toHaveUrl('https://juvi-megarocket-app.vercel.app/auth/sign-in');
+  });
+
+  it('login member', async ()=>{
+    await LogIn.signInBtnClick();
+    await SignUpMember.signIn("pablomorad@hotmail.com", "Chimpance1");
+    await LogIn.submitBtnClick();
+    await expect (ModalAlert.modalAlertText).toBeDisplayed();
+    await ModalAlert.modalAlertButtonClick();
+    await expect(browser).toHaveUrl('https://juvi-megarocket-app.vercel.app/member');
+  });
+
+  it('navigate on activities', async ()=>{
```

Comment:
i don't know why test this, if the test of activities, it's done in this file in master